cmake/Pure32.cmake

```diff
@@ -0,0 +1,18 @@
+option (ENABLE_PURE32 "Use only 32-bit types or smaller" OFF)
+option (ENABLE_RASPBERRYPI "Build a library version that's safe for the Raspberry Pi 3" OFF)
```

Comment:
Can this be expressed not in terms of the raspberry pi, such that it's usable for a variety of other non-pi 32-bit platforms?

Reply:
Should be ENABLE_VC4CL then.

Reply:
Done.

Note that this ENABLE_VC4CL option is probably a temporary interstitial option. doe300 back on the VC4CL repos is addressing a couple of issues that we think are bugs in the OpenCL compiler. Once those are addressed, looking at the changes here, ENABLE_PURE32 will be necessary and sufficient to support VC4CL. While we're figuring that out, though, this will work.